src/components/atoms/PrivateRoute/PrivateRoute.tsx

```diff
@@ -0,0 +1,14 @@
+import { Redirect, Route, RouteProps } from 'react-router';
+import React from 'react';
+import { fakeAuth } from 'Routes/helpers';
+
+const PrivateRoute = ({ children, ...rest }: RouteProps): JSX.Element => {
+  return (
+    <Route
+      {...rest}
+      render={() => (fakeAuth.isAuthenticated ? children : <Redirect to="/login" />)}
+    />
+  );
```

Comment:
Check if this is working in the same way
```suggestion
fakeAuth.isAuthenticated && return <Route {...rest}>{children}</Route>
return <Redirect to="/login" />
```

Reply:
It does't work at all